sys-process/runit/files/1-2.1.2

```diff
@@ -1,10 +1,11 @@
 #!/bin/sh
 # system one time tasks
 
+source /etc/runit/rc.sh
```

Comment:
This script is `#!/bin/sh` and `source` is a bashism. For POSIX shell you should use `. /etc/runit/rc.sh` instead.

Reply:
how about change /bin/sh to /bin/bash? since these scripts are for gentoo only, bash is always available? and most likely we'll test with bash.

Reply:
Its not my call and I'm not a runit user, but Gentoo allows using a POSIX shell so I personally don't really see the point in forcing bash.

Reply:
To me using bash is okay for these scripts.
But, anyway, I recommend to always use `shellcheck` to lint such scripts (it'll catch bashisms too).